vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/util/extension/CheckoutConfig.kt

```diff
@@ -47,3 +52,40 @@ internal val CheckoutConfig.cityOptions: VGSCheckoutCityOptions
 internal val CheckoutConfig.postalCodeOptions: VGSCheckoutPostalCodeOptions
     get() = this.formConfig.addressOptions.postalCodeOptions
 //endregion
+
+internal fun CheckoutConfig.generateBaseUrl(): String {
```

Comment:
@Endorf I still think this function should be inside the `CheckoutConfig` class, as it's used only there, otherwise, it can be moved to a separate class like UrlBuilder/Helper, etc.

Reply:
I replaced it because the generation of URL is not important for developers. It simplifies the class appearance for developers.